From ce68f0b6e8cc76ebe295c5ca2e3f2cb8952061da Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Thu, 6 Feb 2025 20:29:03 +0100
Subject: [PATCH 05/42] qualcommax: disable IPQ5018 specific symbols

Newly introduced IPQ5018 specific symbols will stop compilation of
ipq807x and ipq60xx subtargets, so make sure to disable them in the
generic config.

Link: https://github.com/openwrt/openwrt/pull/17884
Signed-off-by: Robert Marko <robimarko@gmail.com>

(cherry picked from commit 98d7f4202fe9861fcdc64d694aecf6945bbc7de8)
---
 target/linux/qualcommax/config-6.6 | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/target/linux/qualcommax/config-6.6 b/target/linux/qualcommax/config-6.6
index ac5059682a..de2e9ccccf 100644
--- a/target/linux/qualcommax/config-6.6
+++ b/target/linux/qualcommax/config-6.6
@@ -198,8 +198,10 @@ CONFIG_I2C_QUP=y
 CONFIG_IIO=y
 CONFIG_ILLEGAL_POINTER_VALUE=0xdead000000000000
 CONFIG_INITRAMFS_SOURCE=""
+# CONFIG_IPQ5018_PHY is not set
 CONFIG_IPQ_APSS_6018=y
 CONFIG_IPQ_APSS_PLL=y
+# CONFIG_IPQ_CMN_PLL is not set
 # CONFIG_IPQ_GCC_4019 is not set
 # CONFIG_IPQ_GCC_5018 is not set
 # CONFIG_IPQ_GCC_5332 is not set
@@ -333,6 +335,7 @@ CONFIG_PHY_QCOM_QMP_USB=y
 CONFIG_PHY_QCOM_QUSB2=y
 # CONFIG_PHY_QCOM_SGMII_ETH is not set
 # CONFIG_PHY_QCOM_SNPS_EUSB2 is not set
+# CONFIG_PHY_QCOM_UNIPHY_PCIE_28LP is not set
 # CONFIG_PHY_QCOM_USB_HS_28NM is not set
 # CONFIG_PHY_QCOM_USB_SNPS_FEMTO_V2 is not set
 # CONFIG_PHY_QCOM_USB_SS is not set
@@ -406,6 +409,7 @@ CONFIG_QCOM_NET_PHYLIB=y
 CONFIG_QCOM_PIL_INFO=y
 # CONFIG_QCOM_Q6V5_ADSP is not set
 CONFIG_QCOM_Q6V5_COMMON=y
+# CONFIG_QCOM_Q6V5_MPD is not set
 # CONFIG_QCOM_Q6V5_MSS is not set
 # CONFIG_QCOM_Q6V5_PAS is not set
 CONFIG_QCOM_Q6V5_WCSS=y
@@ -537,6 +541,7 @@ CONFIG_SPARSE_IRQ=y
 CONFIG_SPI=y
 CONFIG_SPI_MASTER=y
 CONFIG_SPI_MEM=y
+# CONFIG_SPI_QPIC_SNAND is not set
 CONFIG_SPI_QUP=y
 CONFIG_SQUASHFS_DECOMP_MULTI_PERCPU=y
 CONFIG_SWIOTLB=y
-- 
2.53.0

